pkg/csi/service/wcp/controller.go

```diff
@@ -210,6 +214,16 @@ func (c *controller) Init(config *cnsconfig.Config, version string) error {
 			return err
 		}
 	}
+	if commonco.ContainerOrchestratorUtility.IsFSSEnabled(ctx, common.PodVMOnStretchedSupervisor) {
+		log.Info("Loading CnsVolumeInfo Service to persist mapping for VolumeID to storage policy info")
+		volumeInfoService, err = cnsvolumeinfo.InitVolumeInfoService(ctx)
+		if err != nil {
+			return logger.LogNewErrorf(log, "error initializing volumeInfoService. Error: %+v", err)
+		}
+		if volumeInfoService != nil {
```

Comment:
Is it possible for volumeInfoService == nil  and err == nil?

Reply:
Looking at the `InitVolumeInfoService` implementation, there is no possibility of `volumeInfoService` to be nil when err = nil. But, I think it's just an additional nil check that got copied over from other places.

Reply:
Done